docs/data/routes/docs/fundamentals/services/dictionary/jss-dictionary-api/en.md

```diff
@@ -0,0 +1,116 @@
+      app. If your Sitecore instance only has 1 JSS App, you can specify the root item ID here;
+      otherwise, the service will attempt to figure out the root item for the current JSS App using GraphQL and app name.
+      rootItemId: '{GUID}'
+      */
+```
+
+2. In the file where you want to fetch the data, import and use your new dictionary service instance: 
+
+```javascript
+import { dictionaryService } from './dictionary-service';
+const language = 'en';
+dictionaryService.fetchDictionaryData(language).then(data => {
+    // do something with the data
+});
+```
+
+## Fetching dictionary data with REST
+
+To invoke the REST Dictionary Service from a JSS application: 
+
+1. In a file `dictionary-service.ts`, create an instance of the `RestDictionaryService` class and provide the configuration object:
+
+```javascript
+import { RestDictionaryService } from '@sitecore-jss/sitecore-jss';
+import config from './config';
+
+export const dictionaryService = new RestDictionaryService({
+  apiHost: config.sitecoreApiHost,
+  apiKey: config.sitecoreApiKey,
+  siteName: config.jssAppName,
+});
+```
+2. In the file where you want to fetch the data, import and use your new dictionary service instance: 
+
+```javascript
+import { dictionaryService } from './dictionary-service';
+const language = 'en';
+dictionaryService.fetchDictionaryData(language).then(data => {
+    // do something with the data
+});
+```
+
+### Using a custom data fetcher
+
+The `RestDictionaryServiceConfig` type accepts a `dataFetcher` property. You can use this property to pass a custom data fetcher to your instance of the Dictionary Service. By default, the JSS REST Dictionary service uses `axios`.
+
+To use a REST dictionary service with a custom data fetcher: 
+
+1. In a file `dictionary-service.ts`, create an instance of the `RestDictionaryService` class and provide the configuration object:
+
+```typescript
+import { AxiosResponse } from 'axios';
+import { 
+  AxiosDataFetcher,
+  RestDictionaryService,
+  DictionaryServiceData
+} from '@sitecore-jss/sitecore-jss';
+import config from './config';
+
+// define your custom data fetcher
+function dataFetcher(url: string, data?: unknown): Promise<AxiosResponse<DictionaryServiceData>> {
+  return new AxiosDataFetcher().fetch(url, data);
+}
+
+export const dictionaryService = new RestDictionaryService({
+  apiHost: config.sitecoreApiHost,
+  apiKey: config.sitecoreApiKey,
+  siteName: config.jssAppName,
+  // provide your custom data fetcher to the service instance
+  dataFetcher
+});
+```
+
+2. Import and use the new dictionary service:  
+```javascript
+import { dictionaryService } from './dictionary-service';
+
+const language = 'en';
+
+dictionaryService.fetchDictionaryData(language).then(data => {
+     // do something with the data
+});
+```
+
+If you'd like to learn more about other translation techniques with JSS, see [Content Translation](/docs/techniques/content-translation).
+
```

Comment:
I think it would be helpful to link to the sample app's implementation of this stuff somewhere on this page. 

Devs don't need to write this code if they are working from the sample app's boilerplate, and I'm not sure if this is obvious.

Reply:
Adding some extra headers for "Basic usage example" and "Sample implementations" (for the sample apps).

Reply:
On second thought, mentioning they don't need to bother with these examples if working based on a sample app is sufficient (for now).